### DISCUSSION TITLE
How to deal with meter that blanks display

### DISCUSSION THREAD

**Discussion by shieldsit** [jomjol/AI-on-the-edge-device#2268] [General]
Hi everyone, 

I have an electric meter that blanks the display then shows the current reading, then blanks the display again.  It repeats this cycle and eventually I end up with NNNNNN's.  So I have to reboot and hope the timing is better and doesn't fade.  I was wondering firstly, does anyone else have a meter like this, and if so, how do you deal with it?  I wish there was a setting that would retake the image if the previous results were all N's.

Thanks for any ideas!

**Comment by caco3** [comment]
you could try to use the https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-checkdigitincreaseconsistency and https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-numbermaxratevalue to make sure it only accepts valid numbers.
